fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/query/Select.java

```diff
@@ -319,4 +348,20 @@ public void addPagination(int offset, int rowsPerPage) {
     public OrderByClause getOrderByClause() {
         return this.orderByClause;
     }
+
+    /**
+     * Add a select to UNION with this query.
+     */
+    public void addUnion(Select union) {
+        this.union = union;
+        this.unionAll = false;
+    }
+
+    /**
+     * Add a select to UNION ALL with this query.
+     */
+    public void addUnionAll(Select unionAll) {
```

Comment:
unionAll is a single value, so I think this should be set, not add.

Reply:
Done.